app/models/customer.rb

```diff
@@ -0,0 +1,12 @@
+class Customer < ApplicationRecord
+  has_many :rentals
+  has_many :videos, :through => :rentals 
```

Comment:
Style: I'd probably update this line to match the ones below.

```suggestion
  has_many :videos, through: :rentals 
```